bundles/core/org.eclipse.smarthome.core.persistence/META-INF/MANIFEST.MF

```diff
@@ -6,8 +6,20 @@ Bundle-Version: 0.8.0.qualifier
 Bundle-ManifestVersion: 2
 Bundle-License: http://www.eclipse.org/legal/epl-v10.html
 Bundle-SymbolicName: org.eclipse.smarthome.core.persistence
-Import-Package: org.eclipse.smarthome.core.items,
+Import-Package: io.swagger.annotations,
+ javax.servlet,
+ javax.servlet.http,
+ javax.ws.rs,
+ javax.ws.rs.core,
+ org.eclipse.smarthome.core.items,
+ org.eclipse.smarthome.core.library.types,
  org.eclipse.smarthome.core.persistence,
- org.eclipse.smarthome.core.types
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.io.rest,
+ org.eclipse.smarthome.ui.items,

```

Comment:
I think this is not used now - it's a hangover from Habmin 1 so will remove. 

Sent from my iPhone

> On 20 Jan 2016, at 13:52, Kai Kreuzer <notifications@github.com> wrote:
> 
> In bundles/core/org.eclipse.smarthome.core.persistence/META-INF/MANIFEST.MF:
> 
> >   org.eclipse.smarthome.core.persistence,
> > - org.eclipse.smarthome.core.types
> > + org.eclipse.smarthome.core.thing,
> > + org.eclipse.smarthome.core.types,
> > + org.eclipse.smarthome.io.rest,
> > + org.eclipse.smarthome.ui.items,
> This dependency from a core bundle to a ui bundle is ugly and it does not seem to be at all required...?
> 
> —
> Reply to this email directly or view it on GitHub.
>